app/Model/CoPetition.php

```diff
@@ -2437,11 +2437,6 @@ public function sendApprovalNotification($id, $actorCoPersonId, $action='approva
     
     $args = array();
     $args['conditions']['CoPetition.id'] = $id;
-/*    $args['contain']['CoEnrollmentFlow'] = array(
-      'CoEnrollmentFlowAppMessageTemplate',
-      'CoEnrollmentFlowDenMessageTemplate',
-      'CoEnrollmentFlowFinMessageTemplate'
-    );*/
```

Comment:
Any reason to commit a commented out block? If so there should be a note.

Reply:
The latest commit does not include this block of code